src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -274,8 +275,12 @@ function escapeQuotesForShell(input: string) {
 	return input.replace(new RegExp(`'`, 'g'), `'\\''`);
 }
 
-export function getFeatureLayers(featuresConfig: FeaturesConfig) {
-	let result = '';
+export function getFeatureLayers(featuresConfig: FeaturesConfig, containerUser: string, remoteUser: string) {
+	let result = `RUN \\
+echo "_CONTAINER_USER_HOME=$(getent passwd ${containerUser} | cut -d: -f6)" >> /tmp/build-features/devcontainer-features.builtin.env && \\
```

Comment:
I wasn't familiar with `getent`. This is it working successfully on an `alpine` container 👍 

<img width="650" alt="image" src="https://user-images.githubusercontent.com/23246594/197002314-d49b233a-baf9-432d-9842-b8acb4a65613.png">

Reply:
Did some testing and it looks like it is working across distributions.